oak-doc/src/site/markdown/query/lucene.md

```diff
@@ -483,14 +487,23 @@ type
   type is not used consistently across various nodes then it would recommended
   to specify the type explicitly.
   A binary is only indexed if there is an associated property `jcr:mimeType`.
+  For binary properties, you do not need to index the property separately. 
+  Binary properties are automatically added to the fulltext index (but only there), 
+  if the node is part of the index, if the node type (or mixin) matches, 
+  if the mime type property is set, and if the mime type is indexed
+  (see the Tika configuration).
 
 propertyIndex
 : Whether the index for this property is used for equality conditions, ordering,
-  and is not null conditions.
+  and is not null conditions. Example query:
```

Comment:
Took me a while to understand what is meant by: `and is not null conditions.` I guess it should be read as "and `is-not-null` conditions", right?

Reply:
Yes. I'll quote `is not null` (that is the syntax).